chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -139,8 +143,13 @@ async def instantiate_mempool_manager(
     block_timestamp: uint64 = TEST_TIMESTAMP,
     constants: ConsensusConstants = DEFAULT_CONSTANTS,
     max_tx_clvm_cost: Optional[uint64] = None,
+    get_unspent_lineage_info_for_puzzle_hash: Callable[
+        [bytes32], Awaitable[Optional[UnspentLineageInfo]]
+    ] = zero_calls_get_unspent_lineage_info_for_puzzle_hash,
```

Comment:
this doesn't look necessary

Reply:
Please see my rationale above and let me know what you think.